src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -430,6 +430,9 @@ def default_api_version(self):
 # We should avoid using ad hoc API versions,
 # use the version in a profile as much as possible.
 AD_HOC_API_VERSIONS = {
+    ResourceType.MGMT_APPSERVICE: {
+        'app_service_certificate_orders': '2022-09-01'
+    },
```

Comment:
Please add a comment showing where it is used, such as 

https://github.com/Azure/azure-cli/blob/08e55c47ff77e2aa8f87cfd18c5d3bcaa52d2c9d/src/azure-cli-core/azure/cli/core/profiles/_shared.py#L434